src/sentry/static/sentry/app/views/projectInstall/gettingStarted.jsx

```diff
@@ -14,7 +15,7 @@ class GettingStartedBody extends React.Component {
   render() {
     let {project, organization} = this.context;
     return (
-      <div className="getting-started">
```

Comment:
did we check to see if this was used anywhere?

Reply:
The getting-started class wasn't even declared anywhere.

Reply:
The only hits are in the red side of this diff.